src/uct/ib/ud/base/ud_ep.c

```diff
@@ -1076,17 +1082,17 @@ void uct_ud_ep_pending_purge(uct_ep_h ep_h, uct_pending_purge_callback_t cb,
         ucs_arbiter_group_push_elem(&ep->tx.pending.group,
                                     &ep->tx.pending.elem);
         ucs_arbiter_group_schedule(&iface->tx.pending_q, &ep->tx.pending.group);
-    } 
+    }
     uct_ud_leave(iface);
 }
 
 void  uct_ud_ep_disconnect(uct_ep_h ep)
 {
     /*
-     * At the moment scedule flush and keep ep
+     * At the moment schedule flush and keep ep
```

Comment:
please split all style / type fixes (which are very good to have, btw) to separate PRs
would make code review much easier